src/main/java/Duke.java

```diff
@@ -1,10 +1,147 @@
+import java.io.*;
+import java.util.ArrayList;
+
 public class Duke {
+
+    private final static String filepath =
+            "..\\..\\..\\duke.txt";
+
     public static void main(String[] args) {
+
+        ArrayList<Task> tasks = new ArrayList<Task>();
+
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        // Default messages in Duke
+        String line = "    ____________________________________________________________";
+        String greeting = "     Hello! I'm Duke\n" + "     What can I do for you?";
+        String add = "     added: ";
+        String exit = line + "\n" + "     Bye. Hope to see you again soon!" + "\n" + line;
+        String list_Header = "     Here are the tasks in your list:";
+        String task_Done_Message = "     Nice! I've marked this task as done:";
+        String task_Add_Message = "     Got it. I've added this task:";
+        String delete_Message = "     Noted. I've removed this task:";
```

Comment:
Since all the strings are constant, it would be best to make them `final` and [correspondingly rename them to convention](https://nus-cs2103-ay1920s2.github.io/website/coding-standards/java/intermediate.html#naming), e.g. MESSAGE_DELETE, MESSAGE_GREETING.